src/components/spaces/show/index.js

```diff
@@ -121,6 +124,19 @@ export default class SpacesShow extends Component {
     this.props.dispatch(spaceActions.destroy(this.props.denormalizedSpace))
   }
 
+  onImportSlurp(slurpObj) {
+    const space = this.props.denormalizedSpace
+
+    const spaceUpdates = parseSlurp(slurpObj, space)
+    let noGraphUpdates = {}
+    if (!space.name) {noGraphUpdates.name = spaceUpdates.name}
+    if (!space.description) {noGraphUpdates.description = spaceUpdates.description}
+    this.props.dispatch(spaceActions.update(this._id(), noGraphUpdates))
```

Comment:
I imagine that this dispatch should only be done if noGraphUpdates is not empty.